Pods/AppDevSettings/AppDevSettings/Classes/SettingsViewController.swift

```diff
@@ -0,0 +1,142 @@
+//
+//  SettingsViewController.swift
+//  AppDevSettings
+//
+//  Created by Drew Dunne on 4/11/19.
+//
+
+import SnapKit
+import UIKit
+
+public class SettingsViewController: UITableViewController {
+
+    private var config: SettingsConfig!
+
+    private enum CellIdentifiers {
+        public static let booleanCell = "booleanCell"
+        public static let buttonCell = "buttonCell"
+        public static let inputCell = "inputCell"
+    }
+
+    public init(config: SettingsConfig) {
+
+        super.init(style: .grouped)
+
+        self.config = config
+
+        let headerImageBuffer: CGFloat = 20
+
+        if let image = config.headerImage, let h = config.headerHeight {
```

Comment:
Can we use height instead of h just so it's clear?

Reply:
These are in the Pods' files so I'll have to update the pod as I can't edit these directly.